tests/src/test/scala/org/apache/spark/sql/rapids/ParquetFormatScanSuite.scala

```diff
@@ -1166,7 +1168,7 @@ class ParquetFormatScanSuite extends SparkQueryCompareTestSuite with Eventually
           })
 
           withResource(cudf.Table.readParquet(new File(testPath))) { table =>
-            GpuColumnVector.debug("DIRECT READ", table)
+            debugPrinter.debug("DIRECT READ", table)
```

Comment:
These prints seems like debug code that was accidentally left in the test, and we normally don't have tests emit debug output as a side-effect of running the tests.

Reply:
it was added in #6914. @revans2 , is it ok to remove the stderr  output here?

Reply:
I don't care if we keep it or delete it. The data is small and you occasionally see it show up in the logs, but it is not much data either way.

Reply:
I'll keep it for now to avoid going through the CI again. We can piggyback cleanup onto another PR